app/views/full-page-examples/search/styles.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../../../src/govuk/base";
+
+.full-page-example__search {
+  $govuk-new-link-styles: true;
+  @import "../../../../src/govuk/core/all";
+
+  .app-document-list {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .app-document-list > li {
+    padding-top: govuk-spacing(4);
+    padding-bottom: govuk-spacing(2);
+    border-bottom: 1px solid $govuk-border-colour;
+  }
+
+  .app-metadata-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .app-metadata-list > li {
+    display: inline-block;
+  }
+
+  .app-filter li {
+    padding-bottom: govuk-spacing(4);
+    border-bottom: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-fieldset__legend {
+    margin-top: govuk-spacing(4);
+    margin-bottom: govuk-spacing(4);
+    font-weight: normal;
+  }
+
+  .app-organisation-checkboxes {
+    padding-bottom: govuk-spacing(4);
+    border-bottom: 1px solid govuk-colour("mid-grey");
+  }
+
+  .app-search-result-list__children {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 15px;
+    list-style: none;
+  }
+
+  .app-search-results-summary {
+    position: relative;
+  }
+
+  .app-search-results-summary .subscribe-link {
+    position: absolute;
+    top: 5px;
+    right: 0;
+  }
+
+  .app-search-result-sort {
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .app-search-result-sort label {
+    display: inline;
+    margin-right: govuk-spacing(1);
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .app-next-page__number {
+    text-decoration: underline;
```

Comment:
The use of `text-decoration` here is resetting the `text-decoration-thickness` property to `initial`, so the underline isn't 3px thick on hover like the other links.